sample-projects/sfm3019-arduino/sfm3019/sfm3019.ino

```diff
@@ -0,0 +1,97 @@
+#include "sfm3019.h"
+
+int error = 0;
+SfmConfig sfm3019;
+
+void setup() {
+    Serial.begin(9600);
+    const char* driver_version = sfm_common_get_driver_version();
+    if (driver_version) {
+        Serial.print("\nSFM driver version: ");
+        Serial.println(driver_version);
+    } else {
+        Serial.println("fatal: Getting driver version failed");
+        error = -1;
```

Comment:
Do we need to set error? It's never read until it's written to in L19

Reply:
Fixed.